zerver/lib/bugdown/__init__.py

```diff
@@ -940,6 +949,8 @@ def extendMarkdown(self, md, md_globals):
             '_begin')
         md.inlinePatterns.add('usermention', UserMentionPattern(mention.find_mentions), '>backtick')
         md.inlinePatterns.add('emoji', Emoji(r'(?<!\w)(?P<syntax>:[^:\s]+:)(?!\w)'), '_end')
+        md.inlinePatterns.add('unicodeemoji', UnicodeEmoji(ur'(?<!\w)(?P<syntax>[\U0001F300-\U0001F64F\U0001F680-\U0001F6FF\u2600-\u26FF\u2700-\u27BF])(?!\w)'), '_end')
```

Comment:
Great, thanks!


        -Tim Abbott

On Fri, Jul 1, 2016 at 5:33 PM, Eklavya Sharma <notifications@github.com>
wrote:

> In zerver/lib/bugdown/__init__.py
> <https://github.com/zulip/zulip/pull/1115#discussion_r69369220>:
>
> > @@ -940,6 +949,8 @@ def extendMarkdown(self, md, md_globals):
> >              '_begin')
> >          md.inlinePatterns.add('usermention', UserMentionPattern(mention.find_mentions), '>backtick')
> >          md.inlinePatterns.add('emoji', Emoji(r'(?<!\w)(?P<syntax>:[^:\s]+:)(?!\w)'), '_end')
> > +        md.inlinePatterns.add('unicodeemoji', UnicodeEmoji(ur'(?<!\w)(?P<syntax>[\U0001F300-\U0001F64F\U0001F680-\U0001F6FF\u2600-\u26FF\u2700-\u27BF])(?!\w)'), '_end')
>
> Opened pull request #1160 <https://github.com/zulip/zulip/pull/1160> to
> fix this.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/zulip/zulip/pull/1115/files/6cc3b8a0a1154ee8586c57521de4c8870bf8b1c0#r69369220>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACnm2uakQmS0UX2EvabdwteqZy_5ua4bks5qRbHfgaJpZM4I-Hl3>
> .
>